### PR TITLE
Correct README links

### PR DESCRIPTION
**Pull request by faebebin**
Rename docs-old dir path name and links used in REAMDE.md.

### CODE REVIEW

**Comment by faebebin** [comment]
All of this is probably WIP. But still I think it is good to have working README also on `develop` branch.

File: README.md [hunk 1]
```diff
@@ -7,24 +7,31 @@
...
 
-[<img src="./docs-old/assets/project-view.gif" />](./docs-old/assets/project-view.gif)
-
 This is Free and Open Source Software. You are welcome to use the code and set up your own instance. The Tasking Manager has been initially designed and built by and for the [Humanitarian OpenStreetMap Team](https://www.hotosm.org/), and is nowadays used by many communities and organizations.
 
 ## Get involved!
 
-* Start by reading our [Code of conduct](./docs-old/code_of_conduct.md)
-* Get familiar with our [contributor guidelines](./docs-old/contributing.md) explaining the different ways in which you can support this project! We need your help!
+* Start by reading our [Code of conduct](https://www.hotosm.org/code-of-conduct)
```
Comment (Aadesh-Baral) [inline README.md:14]:
```suggestion
* Start by reading our [Code of conduct](./docs-old/code_of_conduct.md)
```

File: README.md [hunk 2]
```diff
@@ -7,24 +7,31 @@
...
 * Help us and submit [pull requests](https://github.com/hotosm/tasking-manager/pulls)
 
+## Sysadmins
+
+* [architecture](./docs/sysadmins/architecture.md)
+* [ci-cd](./docs/sysadmins/ci-cd.md)
+* [deployment](./docs/sysadmins/deployment.md)
+* [monitoring-logging](./docs/sysadmins/monitoring-logging.md)
+* [networking-connectivity](./docs/sysadmins/networking-connectivity.md)
+* [security-reliability-processes](./docs/sysadmins/security-reliability-processes.md)
```
Comment (Aadesh-Baral) [inline README.md:33]: Can we include this section in the `Setup the TM for development` document rather than the Readme file?

**Review by Aadesh-Baral — CHANGES_REQUESTED** [review]

File: docs/developers/development-setup.md [hunk 3]
```diff
@@ -254,3 +254,13 @@ export TM_DB=postgresql://hottm:hottm@localhost/tasking-manager
...
 Clone the Tasking Manager repository and use `docker-compose up` to get a working version of the API running.
+
+## Sysadmins
+
+* [architecture](./docs/sysadmins/architecture.md)
+* [ci-cd](./docs/sysadmins/ci-cd.md)
+* [deployment](./docs/sysadmins/deployment.md)
+* [monitoring-logging](./docs/sysadmins/monitoring-logging.md)
+* [networking-connectivity](./docs/sysadmins/networking-connectivity.md)
+* [security-reliability-processes](./docs/sysadmins/security-reliability-processes.md)
```
Comment (Aadesh-Baral) [inline docs/developers/development-setup.md:265]:
```suggestion
## Sysadmins guide

* [System architecture](../sysadmins/architecture.md)
* [Managing CI/CD with CircleCI](../sysadmins/ci-cd.md)
* [Deployment Guide](../sysadmins/deployment.md)
```

Comment (Aadesh-Baral) [reply docs/developers/development-setup.md:265]: Fix broken links and remove docs with no content

**Review by Aadesh-Baral — CHANGES_REQUESTED** [review]

File: docs/developers/development-setup.md (hunk 3 above)
Comment (faebebin) [reply docs/developers/development-setup.md:265]: oops, sorry for broken change. Thanks for fixing

**Review by Aadesh-Baral — APPROVED** [review]

**Pull request merged**